tests/helpers.py

```diff
@@ -170,6 +170,7 @@ def setup_dummy_suppliers(self, n):
                     organisation_size='small',
                     duns_number='{}'.format(100000000 + i),
                     registration_country='country:GB',
+                    other_company_registration_number='555-222-111'
```

Comment:
🤔 Do we not set up a `companies_house_number` here?

Reply:
Nope, never been done, can do it as part of this though 🤷‍♂ wasn't too big a deal to add the `other_company_registration_number `

Reply:
Cool - I'll leave it up to you.